src/tests/suite/daos_nvme_recovery.c

```diff
@@ -134,6 +134,12 @@ nvme_recov_1(void **state)
 	}
 	print_message("All targets are in UPIN\n");
 
+	if (mode == 0) {
+		print_message("Disconnect the pool for offline failure\n");
+		rc = daos_pool_disconnect(arg->pool.poh, NULL);
+		assert_rc_equal(-DER_BUSY, rc);
```

Comment:
This pool disconnect failed with EBUSY since there is still container opened, so it's actually not offline test.

To enable real offline test, I'm afraid that you have to setup a separate pool/container.

Reply:
oh, so it does not actually disconnect the connection after DER_BUSY. Let me try to use the existing pool/container by closing the container first and later the pool.